src/imageops/sample.rs

```diff
@@ -898,7 +898,6 @@ where
         x if x == 0.0 => 1.0,
         sum => sum,
     };
-    let sum = (sum, sum, sum, sum);
```

Comment:
Interesting tid bit: f32 mutliplication is usually faster than division. It would make sense to calculate `1.0 / sum` once upfront and use that coefficient instead of dividing by `sum` later on.

Reply:
I'm unsure I understand what you're suggesting. I don't see anywhere that `1.0 / sum` is being used here?

(Side note: All of my PRs allow edits by maintainers. So if you ever want to include such changes in my PRs, go ahead.)

Reply:
Just thought I should mention: I took another look at the commit you made.

After thinking about it for a couple moments, it suddenly clicked for me how your suggestion works. And I'm surprised it didn't click for me before.

So I hope I didn't seem dismissive of your suggestion! I just wasn't sure what was meant until now, and figured the best thing to do was to let you know that you could edit the PR yourself.